python/tests/test_n2f_periodic.py

```diff
@@ -49,8 +49,10 @@ def test_nea2far_periodic(self):
                                 sources=sources,
                                 symmetries=symmetries)
 
-            n2f_obj = sim.add_near2far(fcen, 0, 1, mp.Near2FarRegion(center=n2f_pt, size=mp.Vector3(y=sy)), nperiods=10)
-            dft_obj = sim.add_dft_fields([mp.Ez], fcen, 0, 1, center=dft_pt, size=mp.Vector3(y=sy))
+            n2f_obj = sim.add_near2far(fcen, 0, 1, mp.Near2FarRegion(center=n2f_pt, size=mp.Vector3(y=sy)),
+                                       nperiods=10, decimation_factor=1)
+            dft_obj = sim.add_dft_fields([mp.Ez], fcen, 0, 1, center=dft_pt, size=mp.Vector3(y=sy),
+                                         decimation_factor=1)
```

Comment:
Why does this need to be disabled?

Reply:
The change that caused this one test to fail using automatic decimation was introduced in c8af01d:
```
< decimation_factor = 1/(dt*(freq_max + src_freq_max));

> decimation_factor = int(std::ceil(1/(dt*(freq_max + src_freq_max))));
```
I verified that this change caused the `decimation_factor` to *increase by one* in the failing test. (I also confirmed that among these two DFT monitors `add_near2far` and `add_dft_fields`, only the `add_near2far` monitor needs to have  automatic decimation disabled or its `decimation_factor` to be reduced by one for the test to pass. Automatic decimation can be reinstated for the `add_dft_fields` monitor and the test will pass.)

There are two possible workarounds: (1) we can replace `std::ceil` with `std:floor` to be safe or (2) just continue to disable the automatic decimation for this one unit test as we are doing now.

Reply:
By switching `std::ceil` to `std::floor` when setting the `decimation_factor` in 898d4385, the automatic decimation in `test_n2f_periodic.py` no longer needs to be disabled.